go.mod

```diff
@@ -387,7 +387,7 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
-	gopkg.in/yaml.v3 v3.0.1 // indirect
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
Any reason why the `//indirect` comment was removed?

Reply:
Had to remove indirect here because drone failed repeatedly due to this. After running `go mod tidy` it changed this from indirect to direct, then drone was passing.